br/pkg/restore/systable_restore.go

```diff
@@ -44,15 +44,15 @@ var unRecoverableTable = map[string]struct{}{
 	"schema_index_usage": {},
 }
 
-var sysPrivilegeTableSet = map[string]bool{
-	"user":          true, // since v1.0.0
-	"db":            true, // since v1.0.0
-	"tables_priv":   true, // since v1.0.0
-	"columns_priv":  true, // since v1.0.0
-	"default_roles": true, // since v3.0.0
-	"role_edges":    true, // since v3.0.0
-	"global_priv":   true, // since v3.0.8
-	"global_grants": true, // since v5.0.3
+var sysPrivilegeTableMap = map[string]string{
```

Comment:
I think you can add a comment to describe the meaning of the map value.

Reply:
[fix comments](https://github.com/pingcap/tidb/pull/36652/commits/90dc2b9a4ad8affaaca4258b65ec1a1493138401)